packages/sagas/src/pending/pending.js

```diff
@@ -0,0 +1,62 @@
+import { delay } from 'redux-saga';
+import { call, put, select, take } from 'redux-saga/effects';
+import { api } from '@talend/react-cmf';
+import { Map } from 'immutable';
+
+import { PENDING_DELAY_TO_SHOW, SHOW_PENDING } from '../constants';
+
+const addOrReplace = api.actions.collections.addOrReplace;
```

Comment:
I'm not sure that constant is useful since it's only used a couple times.

Reply:
Other specialized statuses will be added later like SHOW_FETCHING, SHOW_COMPUTING, SHOW_ERROR_STATUS